src/main/webapp/dev/js/common/statusMessage.es6

```diff
@@ -24,6 +24,7 @@ function populateStatusMessageDiv(message, status) {
     $statusMessageDivContent.addClass('alert');
     // Default the status type to info if any invalid status is passed in
     $statusMessageDivContent.addClass(`alert-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
+    $statusMessageDivContent.addClass(`icon-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
```

Comment:
@whipermr5 lint isn't allowing me to reassign the input parameter `status` (as you suggested above) quoting the rule `no-param-reassign`. Would you suggest a new variable to hold this?